src/components/editor/Bindings/FieldSelection/index.tsx

```diff
@@ -87,7 +94,7 @@ const mapConstraintsToProjections = (
         };
     });
 
-function FieldSelectionViewer({ collectionName }: Props) {
+function FieldSelectionViewer({ collectionName, connectorsExist }: Props) {
```

Comment:
Ah, this was something I intended to bring up yesterday. At least one connector must exist in order to render the resource configuration form section, therefore I think it is safe to set the `disabled` property of the `EntityTestButton` component instance explicitly to `false`. This would remove the need for the highlighted `connectorsExist` property entirely. How does that sound to you?

Reply:
That sounds okay. The thing that confused me while skimming the code was the name... I was a bit confused why field selection cared about connectors for the reason you state.

So I guess either add a comment or update the name within here.